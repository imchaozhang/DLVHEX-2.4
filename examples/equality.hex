p(X) :- X==1.
q(X) :- X=1.
