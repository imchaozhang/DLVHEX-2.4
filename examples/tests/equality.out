{q(1), p(1)}
